index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel = "stylesheet" href="css/style.css">
+    <title>Home Page</title>
+</head>
+<body background = "image/images.jpg">
+
+    <form action = "login.html">
```

Comment:
Why use a form here? What are you trying to submit? This can be a stand-alone link